### ISSUE TITLE
pl.read_excel should have a skip_rows argument for "openpyxl" engine

### ISSUE THREAD

**Issue by ldacey** [pola-rs/polars#13879]
### Description

[example-skip-rows.xlsx](https://github.com/pola-rs/polars/files/13999267/example-skip-rows.xlsx)

I attached an example file that I needed to use pandas instead of polars for.

If I use the default engine with read_csv_options to skip rows, the issue is that the HH:MM:SS columns are truncated somehow. 

`pl.read_excel(path, read_csv_options={"skip_rows": 9}` (for some reason we only skip 9 rows with this engine instead of 10 rows?)

For example, 08:45:13 is only 08:45 when the data is read into memory. I need to convert these values to seconds so I can't lose informations. Also, the timestamp/date columns are read as text whereas "openpyxl" reads the dates natively - not a huge deal since I can parse these. Here is a screenshot of the data using "read_csv_options":

<img width="553" alt="image" src="https://github.com/pola-rs/polars/assets/12517855/297e92e7-3c66-454a-823c-07828364a1c0">

If I read the file with `pl.read_excel(path, engine="openpyxl")` the dataframe only includes the summary table up top. This is a system generated file that I need to ingest. Here is a screenshot:

<img width="285" alt="image" src="https://github.com/pola-rs/polars/assets/12517855/0eb9a3e2-4905-4752-9265-4ef6422b9cd6">

Ultimately, the only way to get the data in the format I need was to use `pd.read_excel(path, skip_rows=10, dtype=str)` and convert that to polars.  (I have to use dtype=str or else the columns are inferred to be `pl.Time` but they can exceed 24 hours, the actual data is more of a duration type in HH:MM:SS format). Ideally, I would like to use polars from end to end.

**Comment by alexander-beedie** [comment]
Yup, we should look to better unify the common/generic options for _all_ of the supported engines 🤔 
I'll add it to the pile!

**Comment by ldacey** [comment]
Is it an xlsx2csv bug that it is somehow truncating the seconds values from those strings and only return HH:MM? It seems like weird behavior.

**Comment by ldacey** [comment]
@alexander-beedie 

```
df = pl.read_excel(
    "example-skip-rows.xlsx",
    engine="calamine",
    engine_options={"skip_rows": 10},
)

TypeError: read_excel() got an unexpected keyword argument 'skip_rows'
```

Is this just passing the arguments to the https://github.com/ToucanToco/fastexcel/blob/main/python/fastexcel/_fastexcel.pyi library? It looks like skip_rows and header_row are supported which I expect should resolve this issue for me.

**Comment by PrettyWood** [comment]
@ldacey we released fastexcel 0.9.0 that uses a bigger sample to determine the schema of the columns. Does everything work as expected? If not do not hesitate to open a new issue on fastexcel side with an example file and I'll have a look

**Comment by ldacey** [comment]
Nice - it works as expected for most files. I ran into an issue just now while testing though. I'll comment on the fastexcel original issue right now with a sample.
